modules/afsocket/afinet-dest.c

```diff
@@ -50,6 +50,8 @@
 #  define _GNU_SOURCE 1
 #endif
 
+static const gint MAX_UDP_PAYLOAD_SIZE = 65000;
```

Comment:
max datagram is `65535 - (IP + UDP header)`, not considering jumbograms 👍🏻

Reply:
If the user comes close to this number they should just adjust it acknowledging that they know what they are doing. Is it okay if we leave it just that or you want to set exactly the limit ?

Reply:
TIL: I didn't know that ipv6 supported datagrams above 64k (up to 4GiB-1, a bit crazy)

In any case UDP on top of ipv6 would only support up to 64k as the UDP header contains a 16 bit value of the same.

I would personally either default to 1024 (which would probably be unexpected, but that's the limit in the RFC), or the exact value. 65000 is difficult to reason about.

Reply:
65507 it is.

Reply:
Thanks.